iphone/TitaniumKit/TitaniumKit/Sources/API/TiBlob.m

```diff
@@ -136,6 +137,23 @@ - (id)initWithImage:(UIImage *)image_
   return self;
 }
 
+- (id)initWithSystemImage:(NSString *)imageName
+{
+  if (![TiUtils isIOSVersionOrGreater:@"13.0"]) {
+    return nil;
+  }
+
+  if (self = [super init]) {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
```suggestion
#if IS_SDK_IOS_13
```

I feel a little uncomfortable only wrapping the image creation in the iOS 13 condition. Sure, this code block should never be reached when not built with iOS 13 SDK but only wrapping that single line inside it calls for unexpected behavior in the following lines, e.g. when module developers use `TiBlob` and wrongfully use `initWithSystemImage:`.

Reply:
Agree. Wrapped whole function inside macro.

Macro IS_SDK_IOS_13 will not be available inside TitaniumKit framework.